### PR TITLE
msglist: Respect user's display_emoji_reaction_users setting

### PR DESCRIPTION
**Pull request by chrisbobbe**
Fixes: #5348

### CODE REVIEW

**Comment by chrisbobbe** [comment]
When there are many reactions, it looks the same as before:



Few reactions, including self:




Few reactions, not including self:

(Something seems odd about the wrapping; we can discuss. Not enough space between lines of text?)

**Comment by chrisbobbe** [comment]
Oh, and here's one where there are just a few votes but they're spread out among multiple emoji:

File: src/webview/html/message.js [hunk 1]
```diff
@@ -52,33 +77,45 @@ const messageReactionAsHtml = (
...
+  _: GetText,
 ): string => {
-  const { ownUser } = backgroundData;
+  const { ownUser, displayEmojiReactionUsers } = backgroundData;
 
   if (reactions.length === 0) {
     return '';
   }
+
+  // See the web app's get_vote_text in its reactions.js.
```
Comment (gnprice) [inline src/webview/html/message.js:99]: Hmm, I'm not finding a function by that name.

Comment (gnprice) [reply src/webview/html/message.js:99]: Ah, I see, it's added by https://github.com/zulip/zulip/pull/21013 .

**Review by gnprice — COMMENTED** [review]
Thanks @chrisbobbe! This code all looks good.

> (Something seems odd about the wrapping; we can discuss. Not enough space between lines of text?

Yeah. Try increasing `line-height`? It looks like just on `.reaction` it's set to `1rem`, which is awfully tight.


> Still, better than what the web app does with narrow screens, I think; see [discussion](https://chat.zulip.org/#narrow/stream/6-frontend/topic/reacting-users.20names.3A.20text.20wrapping/near/1392792).)

Heh, yeah -- good find.

**Comment by chrisbobbe** [comment]
Here's with `line-height: 1.2rem`:

File: src/webview/html/message.js (hunk 1 above)
Comment (chrisbobbe) [reply src/webview/html/message.js:99]: Ah, yeah, I was looking at the `chat.zulip.org` branch when putting this together.

**Comment by chrisbobbe** [comment]
Thanks for the review! Revision pushed.

**Comment by gnprice** [comment]
Thanks! Looks good; merging.

**Pull request merged**